### ISSUE TITLE
dzsave does only one TileGroup for zoomify

### ISSUE THREAD

**Issue by tvngithub** [jcupitt/libvips#585]
Hi John,
Thank you so much for super useful VIPS tools!

I am having a problem using dzsave in zoomify format in vips 8.2.2 on Ubuntu 16.04 and I am not sure whether the command arguments have changed. I did try When performing

**%vips dzsave ScannedSlide.mrxs[autocrop] ScannedSlideZ --layout zoomify**

the conversion generates only one directory, the TileGroup0, with few jpg files and the ImageProperties.xml files. You would expect that the conversion generates a complete pyramid of the MRXS file with many TileGroupX directories. In Ubuntu 14.04 I used vips version 7.4 and did a vips extract_area and gave it the bounds from the output of the command **openslide-show-properties ScannedSlide.mrxs**. For some reason this doesn't work anymore due to the error:
extract_area: bad extract area

In short my issue right now is to get the command above with autocrop to generate the whole slide and full pyramid instead of just one subdirectory TileGroup0.

Thank you very much.
tom

**Comment by tvngithub** [comment]
Forgot to mention that the argument **--depth** does only affect slightly the number of tiles generated in the output directory from 32 to 40 (**NUMTILES** variable in the ImageProperties.xml). Only a single subdirectory **TileGroup0** is generated regardless of the **--depth**  option.

**Comment by bgilbert** [comment]
Ensure that VIPS was built with OpenSlide support.  The `.mrxs` file itself is a JPEG containing a small thumbnail of the slide, so if OpenSlide support is not enabled, the thumbnail will be tiled instead.

**Comment by tvngithub** [comment]
This makes a lot of sense. Thanks Benjamin. I am not sure why the default installation of VIPS on Ubuntu 16.04 is not built with OpenSlide support. The tools under Ubuntu 14.04 used OpenSlide as default. Can I get VIPS to use OpenSlide via arguments or options or do I have to build a special version that depends on OpenSlide?
Thanks.
tom

**Comment by tvngithub** [comment]
Thanks again, Benjamin. I installed VIPS built by Dariusz Duma with
sudo add-apt-repository ppa:dhor/myway
and VIPS now does the full conversion as expected. I am closing this thread.

Thanks again, Benjamin and John for the great tools for the community.

**Comment by bgilbert** [comment]
@tvngithub Glad you got it working.  It appears that the 16.04 packages [*are* built with OpenSlide](http://packages.ubuntu.com/xenial/libvips42), so I'm not sure why you were seeing that behavior.
